### ISSUE TITLE
Fails to build when using react-native bundler

### ISSUE THREAD

**Issue by ArnaudRinquin** [lingui/js-lingui#116]
Hi!

I've be trying to use lingui within a react-native project but I have an issue using the `lingui-react` babel. Running `react-native start` will fail with the following error, which is not really helping:

```
error: bundling failed: TypeError: /Users/arnaud/projects/slite/mobile/node_modules/lingui-react/dist/index.js: undefined is not iterable!
    at module.exports.require.getIterator (/Users/arnaud/projects/slite/mobile/node_modules/core-js/library/modules/core.get-iterator.js:5:42)
    at processMethod (/Users/arnaud/projects/slite/mobile/node_modules/babel-plugin-lingui-transform-js/dist/index.js:53:30)
```

I think I don't have any error in my setup, which is dead simple:

Relevant deps:

```
"react-native": "0.50.3",
"lingui-i18n": "^1.3.2",
"lingui-react": "^1.3.4",
"babel-preset-lingui-react": "^1.0.9",
"babel-preset-react-native": "4.0.0",
```

Lingui config:

```json
{
  "fallbackLocale": "en",
  "sourceLocale": "en",
  "localeDir": "<rootDir>/src/locale",
  "srcPathDirs": [
    "<rootDir>/src"
  ],
  "srcPathIgnorePatterns": [
    "/node_modules/"
  ],
  "format": "lingui"
}
```

.babelrc:

```
{
  "presets": [
    "react-native",
    "lingui-react"
  ]
}
```

File structure:

<img width="353" alt="screen shot 2017-11-13 at 17 21 04" src="https://user-images.githubusercontent.com/228886/32736224-3260f7f4-c897-11e7-8d11-50e4ab39f223.png">

The build works fine when I remove `lingui-react` from `.babelrc` but then I get the fair warning:

<img width="343" alt="screen shot 2017-11-13 at 17 31 32" src="https://user-images.githubusercontent.com/228886/32736849-ce230b22-c898-11e7-97cd-d6ccbdf49bfd.png">

**Comment by tricoder42** [comment]
Hi, I’m off until Wednesday. I’ll let you know once I’m back online 👍

13. 11. 2017 v 17:23, Arnaud Rinquin <notifications@github.com>:

**Comment by ArnaudRinquin** [comment]
I recreated [a repo](https://github.com/ArnaudRinquin/rntest) so you can quickly replicate the issue.

The config is even more basic than the one I exposed above.

**Comment by tricoder42** [comment]
@ArnaudRinquin Thank you, that was super helpful!

**TL;DR:** I managed to track down this obscure bug and I've just released fix in **lingui-react@1.3.5**.

More info: For some reason, babel-plugins proces all source files in node_modules. This happens in react-native environment only, I haven't experienced it in webpack. When `lingui-transform-js` plugin transforms `lingui-react` lib, it finds `i18n.select` call inside `Select` component and tries to transform it to `i18n._('{variable, select, ...}'), which is wrong. So I aliased `i18n` object to `_i18n` to trick lingui plugin. Hopefully, I'll find better solution later.

**Comment by tricoder42** [comment]
@ArnaudRinquin Please let me know if it works for your project as well. I'm able to run your testing repo without any errors.

**Comment by ArnaudRinquin** [comment]
I just tried it: 👌 Thanks @tricoder42 for the quick fix! 

Awesome lib by the way: super simple yet comprehensive API and exactly the right CLI tooling we need. This project deserves more exposure!

**Comment by tricoder42** [comment]
Great, thank you! 👍 Getting exposure is hard, but people are starting using it, talking about it and writing blog posts, so it should get better :)

If you wish I could mention your project/company in README to get a bit of social proof
